examples/game-of-life/build/untouched.wat

```diff
@@ -60,7 +60,7 @@
        global.get $assembly/config/BGR_DEAD
        i32.const 16777215
        i32.and
-      else       
```

Comment:
I used the automated update feature and it did this. I'm guessing perhaps these files were generated by manual copy/paste or by hand previously maybe?

Reply:
Looks like the output is from an earlier version of Binaryen. It used to emit such whitespace and I guess the example fixtures haven't been updated since then. Most likely nothing to worry about :)